lib/socket.js

```diff
@@ -65,7 +65,7 @@ class Transport extends EventEmitter {
     this._buffer += chunk;
 
     try {
-      this._buffer = jsrs.parseNetworkPackets(this._buffer, packets);
+      this._buffer = json5.parseNetworkPackets(this._buffer, packets);
```

Comment:
@nechaido, as @aqrln said, you can't simply split this into two addons, because message parsing is tightly coupled with internals of our parser.